components/Dialogs/src/MDCAlertController.m

```diff
@@ -687,7 +720,11 @@ - (void)setupAlertView {
     self.alertView.titleLabel.adjustsFontForContentSizeCategory =
         self.adjustsFontForContentSizeCategory;
   }
-  self.alertView.messageLabel.text = self.message;
+  if (self.attributedMessage.count > 0) {
```

Comment:
Looking at this again, I think we can/should just rely on UILabel handling the fallback behavior for us and that we should just set both properties unconditionally here.

Reply:
So, I'd better just test against `nil` to see which property I set (`message` vs `attributedMessage`). Got it

Reply:
Ah sorry, to be more clear:

```
self.alertView.messageLabel.text = messageText
self.alertView.messageLabel.attributedText = messageAttributedText
```

Just set both directly and we can let UILabel decide what to do (I believe it prioritizes the attributed text).

Reply:
The thing is that whenever the `attributedText` property is set, it replaces the `text` property value with the string data (without formatting) of the attributed string. So I'm afraid if we follow that approach the `message` property will be always shadowed by the `attributedString` property.

Here's a [link](https://developer.apple.com/documentation/uikit/uilabel/1620542-attributedtext) to this behavior in the Apple Docs

Reply:
Ah you're right, thank you for checking! This LGTM then :)